src/main/resources/templates/rust/test/tests/structural_helpers/mod.rs

```diff
@@ -0,0 +1,173 @@
+#![allow(dead_code)]
+
+use std::io::Read;
+
+use syn::{
+    Fields, ImplItem, ImplItemFn,
+    Item::{self, Impl, Struct, Trait},
+    ItemImpl, ItemStruct, ItemTrait, TraitItem, TraitItemFn,
+};
+
+pub fn check_struct_names<'a, I: IntoIterator<Item = &'a str>>(items: &[Item], names: I) {
+    for name in names {
+        find_struct(items, name).unwrap();
```

Comment:
```suggestion
        find_struct(items, name).expect(format!("Could not find struct {}", name));
```
Maybe a better error message would be helpful?

Reply:
Yes that's true. But I think the module user should specify the error message. Addressed in https://github.com/ls1intum/Artemis/pull/8802/commits/50a94acc1f42823ddd7f6808484c3f622a7f8c2b.